app/src/main/java/com/morihacky/android/rxjava/fragments/ConcurrencyWithSchedulersDemoFragment.java

```diff
@@ -64,6 +64,10 @@ public View onCreateView(LayoutInflater inflater,
     @OnClick(R.id.btn_start_operation)
     public void startLongOperation() {
 
+        if(_subscription!=null){
```

Comment:
is there a specific case that was causing problems, that prompted you to add this here?

Also there's a `RxUtils.unsubscribeIfNotNull(_subscription);` helper